### PR TITLE
docs: update example output in HTTP-aware policy enforcement GSG

### PR DESCRIPTION
**Pull request by tklauser**
Update the example output of the commands to the way they look wiht
Cilium 1.8.0-rc3.

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]

File: Documentation/gettingstarted/http.rst [hunk 1]
```diff
@@ -324,17 +335,36 @@ You can observe the L7 policy via ``kubectl``:
     Name:         rule1
     Namespace:    default
     Labels:       <none>
-    Annotations:  kubectl.kubernetes.io/last-applied-configuration:
-                    {"apiVersion":"cilium.io/v2","description":"L7 policy to restrict access to specific HTTP call","kind":"CiliumNetworkPolicy","metadata":{"...
-    API Version:  cilium.io/v2
+    Annotations:  API Version:  cilium.io/v2
```
Comment (joestringer) [inline Documentation/gettingstarted/http.rst:338]: Is this correct? Previously `Annotations` and `API Version` were on separate lines.

**Review by joestringer — CHANGES_REQUESTED** [review]
Minor nit, otherwise LGTM

File: Documentation/gettingstarted/http.rst (hunk 1 above)
Comment (tklauser) [reply Documentation/gettingstarted/http.rst:338]: Good question :thinking: this is what I by from running `kubectl describe ciliumnetworkpolicies > cnp.txt` and then pasting the contents of `cnp.txt` here. Will check again.

Comment (tklauser) [reply Documentation/gettingstarted/http.rst:338]: Just checked again on minikube. This is indeed what I get when running `kubectl describe ciliumnetworkpolicies`:

```
Name:         rule1
Namespace:    default
Labels:       <none>
Annotations:  API Version:  cilium.io/v2
Description:  L7 policy to restrict access to specific HTTP call
Kind:         CiliumNetworkPolicy
Metadata:
  Creation Timestamp:  2020-06-15T19:07:32Z
  Generation:          2
  Managed Fields:
    API Version:  cilium.io/v2
    Fields Type:  FieldsV1
    fieldsV1:
      f:description:
      f:metadata:
        f:annotations:
          .:
          f:kubectl.kubernetes.io/last-applied-configuration:
      f:spec:
        .:
        f:endpointSelector:
          .:
          f:matchLabels:
            .:
            f:class:
            f:org:
        f:ingress:
    Manager:         kubectl
    Operation:       Update
    Time:            2020-06-15T19:08:15Z
  Resource Version:  1012
  Self Link:         /apis/cilium.io/v2/namespaces/default/ciliumnetworkpolicies/rule1
  UID:               13e03438-c46c-4063-b74b-39240205557b
Spec:
  Endpoint Selector:
    Match Labels:
      Class:  deathstar
      Org:    empire
  Ingress:
    From Endpoints:
      Match Labels:
        Org:  empire
    To Ports:
      Ports:
        Port:      80
        Protocol:  TCP
      Rules:
        Http:
          Method:  POST
          Path:    /v1/request-landing
Events:            <none>
```

Comment (joestringer) [reply Documentation/gettingstarted/http.rst:338]: *shrug*

If that's what it says, that's what it says. I see the same thing.

**Review by joestringer — APPROVED** [review]

**Review by tgraf — APPROVED** [review]

**Pull request merged**